Decred Wallet/Extensions/UIColor.swift

```diff
@@ -10,6 +10,19 @@ import Foundation
 import UIKit
 
 extension UIColor {
+    
```

Comment:
You can get rid of the struct and just have the custom color called statically off the UIColr. e.g UIColor.orangeColor

Reply:
resolved in 01da249